project-code/project/plugins.sbt

```diff
@@ -6,4 +6,6 @@ resolvers += Resolver.typesafeRepo("releases")
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.3.2")
 
-addSbtPlugin("me.lessis" % "bintray-sbt" % "0.3.0")
\ No newline at end of file
+addSbtPlugin("me.lessis" % "bintray-sbt" % "0.3.0")
+
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "4.0.0")
```

Comment:
Can you please remove this plugin from the project code?
For info, you can add it into `~/.sbt/0.13/plugins/sbteclipse.sbt` for example.